<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sachintaw-w10" ignored="0" name="SampleTest" tests="2" failures="1" timestamp="13 Oct 2021 09:01:32 GMT" time="8.378" errors="0">
  <testcase name="testMethod1" time="0.031" classname="com.cybage.assignment.page.sample">
    <failure type="java.lang.AssertionError" message="Method-1 failure expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Method-1 failure expected [true] but found [false]
at com.cybage.assignment.page.sample.testMethod1(sample.java:74)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod2" time="0.0" classname="com.cybage.assignment.page.sample"/>
</testsuite> <!-- SampleTest -->
